src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1486,7 +1486,7 @@ def listen(self):
         """Start listen Redis keyspace events and will trigger corresponding handlers when content of a table changes.
         """
         self.pubsub = self.get_redis_client(self.db_name).pubsub()
-        self.sub_thread = threading.Thread(target=self.listen_thread, args=(0.01,))
+        self.sub_thread = threading.Thread(target=self.listen_thread, args=(10,))
```

Comment:
This issue found by @bluecmd, for more info please check discussion of this PR: https://github.com/sonic-net/sonic-buildimage/pull/13836

Reply:
Thanks for the fix!

I believe the timeout value should be a float according to the documentation I found, but maybe it doesn't matter. It's a `double` in swsscommon but maybe Python converts it.

Reply:
@bluecmd , Can you approve this PR if the fix is ok for you?

Reply:
Done!

Reply:
We should fix the bug inside swss-common, so every application could benefit automatically.

Reply:
I checked SONiC code, here is the only place using get_message() with timeout, so change code inside sonic-swss-common will not break existed code, I will close this PR and create new fix in sonic-swss-common

Reply:
Will close this PR because PR in sonic-swss-common created: https://github.com/sonic-net/sonic-swss-common/pull/864